docs/source/onedal/spmd/index.rst

```diff
@@ -0,0 +1,68 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _dg_spmd:
+
+============================
+Single Program Multiple Data
+============================
+
+This section includes concepts and description of objects that support 
+distributed computations using :capterm:`SPMD` model.
+
+#. Distributed computation using SPMD model
+
+In typical usage scenarios, a user provides a :capterm:`communicator` object as a first parameter of 
+a free function to indicate that the algorithm can process data simultaneously. All internal inter-process 
+communications at sync points required by the algorithm are hidden from the user.
+
+
+.. _example_spmd_flow:
+
+.. figure:: _static/spmd_flow.png
+  :width: 800
+  :alt: Typical SPMD flow
+
+  Example of SPMD Flow in oneDAL
+
+.. _communicator_operations:
+
+Supported Collective Operations
+-------------------------------
+
+The following collective operations are supported:
+
+- Bcast
+  Broadcasts data from specified process.
+
+- Allreduce
+  Reduces data among all processes.
+
+- Allgatherv
+  Gathers data from all processes and share the result among all processes.
+
+.. _backend_restrictions:
+
+Backend specific restrictions
+-----------------------------
+
+#. OneCCL:
```

Comment:
again, this is a numbered list. is it intentional?

`OneCCL` -> `oneCCL`

Reply:
Done